### ISSUE TITLE
Improve contrast of links

### ISSUE THREAD

**Issue by Strappazzon** [Strappazzon/td2-map-brooklyn#2]
### Description

The links color is failing the WCAG contrast check.

### Checklist

- [x] I agree to follow the [Code of Conduct](https://github.com/Strappazzon/.github/blob/-/CODE_OF_CONDUCT.md).
- [x] I have read the [Contributing Guidelines](https://github.com/Strappazzon/td2-map-brooklyn/blob/-/.github/CONTRIBUTING.md).
- [x] I have read the [Support Policy](https://github.com/Strappazzon/td2-map-brooklyn/blob/master/SUPPORT.md#support-policy).
